optaweb-employee-rostering-standalone/pom.xml

```diff
@@ -191,6 +191,8 @@
                     <argument>${project.parent.basedir}/${frontend.project.name}:/e2e:Z</argument>
                     <argument>-w</argument>
                     <argument>/e2e</argument>
+                    <argument>${user.flag}</argument>
+                    <argument>${user.name.group}</argument>
```

Comment:
What are these parameters in the case of podman?

Reply:
Then user.flag and user.name.group are empty so executor plugin skips that and podman launches the same way as before